### DISCUSSION TITLE
VDM support

### DISCUSSION THREAD

**Discussion by rshalit-rs** [openbmc/libmctp#16] [Q&A]
Hello,

Is the library supports VDM?

Thanks

**Comment by amboar** [comment]
VDM in what regard? VDM MCTP messages, or the PCIe VDM MCTP transport binding? The former shouldn't be an issue, but there's no plan to support the latter unless it's for freestanding firmware. Under Linux you should use [AF_MCTP sockets](https://docs.kernel.org/networking/mctp.html) and not libmctp (it's deprecated under Linux), and implement a driver for your PCIe VDM-based MCTP controller.
